src/Smidge/Models/IWebFile.cs

```diff
@@ -7,6 +7,11 @@ namespace Smidge.Models
     public interface IWebFile
     {
         string FilePath { get; set; }
+        /// <summary>
+        /// If the file has different of physical path and request path
+        /// Look at https://github.com/Shazwazza/Smidge/issues/74
+        /// </summary>
+        string RequestPath { get; set; }
```

Comment:
As this is changing an interface, it's a breaking change and I'd have to release a new major version. I don't see where this gets set? is this set manually by the developer? If so, isn't there a nicer way to achieve this so the developer doesn't need to worry about this (i.e. by passing in `StaticFileOptions` during the Smidge/bundle config that were used in configuring static files somehow)

Reply:
An Idea is when you create bundles into IBundleManager, you can create array
Bundle Create(string bundleName, PreProcessPipeline pipeline, params JavaScriptFile[] jsFiles);
and fill RequestPath at same time.